### ISSUE TITLE
New Gamemode - Leadership

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#565]

**Comment by Bear64DX** [comment]
This could replace Sardines' index (real)

**Comment by TheMan6900** [comment]
> This could replace Sardines' index (real)

Nah

**Comment by Bear64DX** [comment]
Would the taggers work together?

**Comment by TheMan6900** [comment]
Oh yeah this would be the replacement for search

**Comment by Bear64DX** [comment]
I'd rather have it replaces Sardines tbh

**Comment by Bear64DX** [comment]
It does sound like a cool gm idea

**Comment by TheMan6900** [comment]
> I'd rather have it replaces Sardines tbh

Arent gamemodes kinda hard to remove though?

**Comment by Bear64DX** [comment]
Yeah they would be. Not sure about index replacing though. I still think they'd be hard to replace, I'm just saying it'd be neat if it did. Plus, there's no reason to replace Search cause like Emerald said, it doesn't have a stats index yet.

**Comment by EmeraldLoc** [comment]
The idea's pretty neat. The runner would take KB, it would just be reduced. To avoid what happened with Sardines and Search, I'd make this gamemode have multiple runners. The name is not very good, so that needs to be rethought of. 

> I'd rather have it replaces Sardines tbh

Sardines is staying, it's been out for an update, it's gonna stay.

> Oh yeah this would be the replacement for search

Yes it would

**Comment by Bear64DX** [comment]
So it'd kinda be like old juggernaut in terms of KB? That's fine by me

**Comment by TheMan6900** [comment]
yeah

**Comment by Bear64DX** [comment]
What about "Throne" as a gm name? (The runners are kinda like kings in this gm if you ask me)

**Comment by TheMan6900** [comment]
hmm

**Comment by TheMan6900** [comment]
Name Ideas: Leadership, Throne, and king

**Comment by EmeraldLoc** [comment]
Don't like any of those names

**Comment by Bear64DX** [comment]
:rip:

**Comment by TheMan6900** [comment]
😢

**Comment by Bear64DX** [comment]
New Ideas: Authority, Dominion, Administration, Governance (shout outs to google for giving me words that mean leadership)

**Comment by TheMan6900** [comment]
YIPPEE
